### DISCUSSION TITLE
Text Font Color

### DISCUSSION THREAD

**Discussion by trace1729** [typst/typst#1111] [Q&A] [answered]
I wonder how can I change the font color

**Comment by Thumuss** [comment, marked as answer]
![Screenshot_2023-05-05-04-08-58-118_org mozilla firefox-edit](https://user-images.githubusercontent.com/42680097/236364545-238188b8-f6ac-4587-a55d-2177a6ff0084.jpg)
Everything is in the docs

**Reply by trace1729** [reply]
Thanks!
